charts/delphai-deployment/templates/deployment.yaml

```diff
@@ -149,6 +151,14 @@ spec:
               value: "{{ $.Release.Name }}-redis-master.{{ $.Release.Namespace }}"
             {{ end }}
 
+            {{ if $.Values.rabbitmq.enabled }}
+            - name: RABBITMQ_CONNECTION_STRING
+              valueFrom:
+                secretKeyRef:
+                  name: rabbitmq
```

Comment:
The k8s secret was created here: https://github.com/delphai/infrastructure/pull/115

Reply:
For mongo we use different approach https://github.com/delphai/helm-charts/blob/0a8d7727d2c5600bc802e0bfa2d44d1807f3192d/charts/delphai-deployment/values.yaml#L82-L85

Lets maybe do the same there to be consistent and keep it simpler?

Reply:
The secret reference moved in 450cee6047673e1e6c1177c268131542488b4be6.